Sources/Public/Animation/LottieAnimationLayer.swift

```diff
@@ -1008,6 +1008,14 @@ public class LottieAnimationLayer: CALayer {
     }
   }
 
+  public func updateRasterizationState() {
```

Comment:
Let's keep this internal

```suggestion
  func updateRasterizationState() {
```

Reply:
Oh, no!

Sorry, I misunderstood that LottieAnimationView was a different module. I fixed it and requested it again!